proto/axelar-cork/v1/cork.proto

```diff
@@ -0,0 +1,47 @@
+syntax = "proto3";
+package axelar_cork.v1;
+
+import "gogoproto/gogo.proto";
+
+option go_package = "github.com/peggyjv/sommelier/x/axelarcork/types";
+
+// todo: do we need Axelar in the name or should we depend on the namespace?
```

Comment:
I'm a fan of the specificity

Reply:
axelarcorktypes.AxelarCork.AxelarTargetContractAddress

Ashes to ashes, java to java